cmd/tailscaled/depaware.txt

```diff
@@ -93,6 +93,7 @@ tailscale.com/cmd/tailscaled dependencies: (generated by github.com/tailscale/de
    L    github.com/google/nftables/expr                              from github.com/google/nftables+
    L    github.com/google/nftables/internal/parseexprfunc            from github.com/google/nftables+
    L    github.com/google/nftables/xt                                from github.com/google/nftables/expr+
+        github.com/google/uuid                                       from tailscale.com/ipn/ipnlocal
```

Comment:
please remove this dep. UUIDs are overused for some random hex bytes and that package isn't worth bringing in for just that. We don't actually need proper UUIDs here.